StreetDrop/StreetDrop/Presentation/Settings/View/SettingHeaderView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SettingHeaderView.swift
+//  StreetDrop
+//
+//  Created by jihye kim on 15/06/2024.
+//
+
+import SnapKit
+import UIKit
+
+final class SettingHeaderView: UICollectionReusableView {
+    private let titleLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupViews() {
```

Comment:
configureUI() 로 통일 부탁드립니다!
```suggestion
    private func configureUI() {
```